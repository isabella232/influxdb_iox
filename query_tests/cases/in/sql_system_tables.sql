-- IOX_SETUP: OldTwoMeasurementsManyFieldsOneRubChunk

--
-- system tables reflect the state of chunks, so don't run them
-- with different chunk configurations.
--


-- ensures the tables / plumbing are hooked up (so no need to
-- test timestamps, etc)
-- IOX_COMPARE: sorted
SELECT partition_key, table_name, storage, memory_bytes, row_count from system.chunks;


-- IOX_COMPARE: sorted
SELECT * from system.columns;
