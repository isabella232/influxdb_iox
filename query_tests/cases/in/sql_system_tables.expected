-- Test Setup: OldTwoMeasurementsManyFieldsOneRubChunk
-- SQL: SELECT partition_key, table_name, storage, memory_bytes, row_count from system.chunks;
-- Results After Sorting
+---------------+------------+-------------------+--------------+-----------+
| partition_key | table_name | storage           | memory_bytes | row_count |
+---------------+------------+-------------------+--------------+-----------+
| 1970-01-01T00 | h2o        | ReadBuffer        | 4536         | 3         |
| 1970-01-01T00 | o2         | OpenMutableBuffer | 1827         | 2         |
+---------------+------------+-------------------+--------------+-----------+
-- SQL: SELECT * from system.columns;
-- Results After Sorting
+---------------+------------+-------------+-------------+---------------+
| partition_key | table_name | column_name | column_type | influxdb_type |
+---------------+------------+-------------+-------------+---------------+
| 1970-01-01T00 | h2o        | city        | String      | Tag           |
| 1970-01-01T00 | h2o        | other_temp  | F64         | Field         |
| 1970-01-01T00 | h2o        | state       | String      | Tag           |
| 1970-01-01T00 | h2o        | temp        | F64         | Field         |
| 1970-01-01T00 | h2o        | time        | I64         | Timestamp     |
| 1970-01-01T00 | o2         | city        | String      | Tag           |
| 1970-01-01T00 | o2         | reading     | F64         | Field         |
| 1970-01-01T00 | o2         | state       | String      | Tag           |
| 1970-01-01T00 | o2         | temp        | F64         | Field         |
| 1970-01-01T00 | o2         | time        | I64         | Timestamp     |
+---------------+------------+-------------+-------------+---------------+
